src/algorithm/ATOMIC-Cuda.cpp

```diff
@@ -289,8 +225,9 @@ void ATOMIC::setCudaTuningDefinitions(VariantID vid)
           if (run_params.numValidAtomicReplication() == 0u ||
               run_params.validAtomicReplication(replication)) {
 
-            addVariantTuningName(vid, "replicate_"+std::to_string(replication)+
-                                      "_global_"+std::to_string(block_size));
+            addVariantTuning<&ATOMIC::runCudaVariantReplicateGlobal<decltype(block_size)::value, replication>>(
+                vid, "replicate_"+std::to_string(replication)+
+                     "_global_"+std::to_string(block_size));
```

Comment:
Here is an example of where we're passing the member function pointer into the template parameter and giving the variant and name as parameters.